src/main/java/ru/spbstu/reader/dto/VcfRecord.java

```diff
@@ -0,0 +1,53 @@
+package ru.spbstu.reader.dto;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Map;
+import java.util.Objects;
+
+@Getter
+@AllArgsConstructor
+public class VcfRecord {
+    private final String chrom;
+    private final int pos;
+    private final String id;
+    private final Allele ref;
+    private final Allele alt;
+    private final Map<String, Object> info;
+
+    public static VcfRecord from(final VariantContext variantContext) {
+        return new VcfRecord(
+                variantContext.getContig(),
+                variantContext.getEnd(),
+                variantContext.getID(),
+                variantContext.getReference(),
+                variantContext.getAlternateAllele(0),
+                variantContext.getAttributes());
```

Comment:
If  we use `final` modifiers it's enough to use `@RequiredArgsConstructor` annotation.